src/Apps/Artist/Components/ArtistCollectionsRail/ArtistCollectionEntity.tsx

```diff
@@ -150,14 +151,18 @@ export const ArtistCollectionEntityFragmentContainer = createFragmentContainer(
         slug
         title
         price_guidance
-        artworks(size: 3, sort: "-decayed_merch") {
-          hits {
-            artist {
-              name
-            }
-            title
-            image {
-              url(version: "small")
+        artworks(aggregations: [TOTAL], sort: "-decayed_merch") {
+          artworks_connection(first: 3) {
+            edges {
+              node {
+                artist {
+                  name
+                }
+                title
+                image {
+                  url(version: "small")
+                }
+              }
```

Comment:
This is the heart of the this change, and is modeled after a known-to-be-working collection → artworks query —namely the artwork grid on the /collection/:foo page, as deduced from the Relay debugger dev tool.

(Just about everything else in this PR is relay compiler / test fixture fallout)